climate-monitoring-client/src/main/java/climatemonitoring/CenterInfo.java

```diff
@@ -1,13 +1,60 @@
 package climatemonitoring;
 
+import climatemonitoring.core.Center;
+import climatemonitoring.core.ConnectionLostException;
+import climatemonitoring.core.DatabaseRequestException;
 import climatemonitoring.core.ViewState;
+import climatemonitoring.core.headless.Console;
 
 class CenterInfo extends ViewState {
 
 	@Override
 	public void onHeadlessRender(String args) {
 
-		throw new UnsupportedOperationException("Unimplemented method 'onHeadlessRender'");
+		try {
+
+			String[] splitargs = args.split(" ");
+			
+			String by = splitargs[0];
+			
```

Comment:
This is ok, but consider using the `climatemonitoring.core.utility.Command` class to split the `<by>` command from the `<args>`